pkg/config/config.go

```diff
@@ -30,12 +30,12 @@ func New() Config {
 		OauthProxyContainerName: getEnv("OAUTH_PROXY_CONTAINER_NAME", "mdc-oauth-proxy"),
 
 		MDCImageStreamName:        getEnv("MDC_IMAGE_STREAM_NAME", "mdc-imagestream"),
-		MDCImageStreamTag:         getEnv("MDC_IMAGE_STREAM_TAG", "master"),
+		MDCImageStreamTag:         getEnv("MDC_IMAGE_STREAM_TAG", "1.1.11"),
```

Comment:
@aliok I think the work that you're doing will collide with this. I think a better solution would be if in your work, when you pin to a version, that we still allow overriding with an unsupported env var. This way we can have the stable thing here by default, and QE can override with the master tag of the operands in whatever testing scenarios that they need to. WDYT?

Reply:
I've just realised that this is _reverting_ the change to `master` tag, nevermind :)